src/uct/rocm/cma/rocm_common.c

```diff
@@ -207,7 +207,11 @@ static hsa_status_t uct_rocm_check_ptr_info(void *ptr, void **gpu_ptr,
     info.size = sizeof(hsa_amd_pointer_info_t);
 
     *need_lock = 1;
-    *gpu_ptr = 0;
+
```

Comment:
need to add mellanox copyright

Reply:
fixed

Reply:
All caller of this function will set gpu_ptr, so this is just a warning of coverity, no real seg fault, right?

Reply:
in this case, why is gpu_ptr checked for NULL later in this function?

Reply:
few lines below there is evaluation for pointer